api/api/utils/image_proxy/__init__.py

```diff
@@ -93,14 +97,33 @@ def _tally_response(
     Pulled into a separate function to help reduce overload when skimming
     the `get` function, which is complex enough as is.
     """
-    tallies.incr(f"thumbnail_response_code:{month}:{response.status}"),
-    tallies.incr(
-        f"thumbnail_response_code_by_domain:{domain}:" f"{month}:{response.status}"
-    )
-    tallies.incr(
-        f"thumbnail_response_code_by_provider:{media_info.media_provider}:"
-        f"{month}:{response.status}"
-    )
+
+    logger = parent_logger.getChild("_tally_response")
+
+    with tallies_conn.pipeline() as tallies:
```

Comment:
I don't think using them as a context manager adds many benefits but maybe for consistency, we could. I hoped it would automatically handle executes (if OK) and rollbacks (if exception) but [seems like that's not the case](https://github.com/redis/redis-py/issues/730#issuecomment-216215345).

Reply:
Fixed in 65f7dba.